content/contribute/code.md

```diff
@@ -0,0 +1,124 @@
+---
+How to contribute to Wazo Platform code base
+---
+
+In order to contribute to Wazo Platform you need to be able to
+retrieve the source code, edit the code, try your changes and contribute
+the code to the Git repository.
+
+# Getting the code
+
+The source code for Wazo Platform is available on
+[GitHub](https://github.com/wazo-platform). Our GitHub organization contains
+over 200 repositories. Finding the one you want to contribute can be a
+daunting task.
+
+The [documentation page](/documentation) can help you find
+which repository you should be working on.
+[Asking for help](#asking-for-help) is always an option when looking at the
+less popular corners of the source code.
+
+You can then
+[clone](https://help.github.com/en/articles/cloning-a-repository) the
+desired repositories on you hard drive and start coding.
+
+# Editing the code
+
+Most of Wazo Platform is written in Python, our code follows the
```

Comment:
`Most of Wazo Platform is written in Python, our code follows the
[PEP8](https://www.python.org/dev/peps/pep-0008/) conventions.`
=> 
`Most of Wazo Platform is written in Python, following the
[PEP8](https://www.python.org/dev/peps/pep-0008/) conventions.`